tests/data/transcription/est00.txt

```diff
@@ -0,0 +1,5 @@
+0.120   0.290   225.000
```

Comment:
It's .m files, so in principle it should be possible. Haven't tried it. The "old" code uses the standard MIREX format, but I think the "new" code uses a custom annotation format. Why?